keyboards/redragon/k556/info.json

```
@@ -0,0 +1,314 @@
+{
+    "keyboard_name": "K556 Devajaras",
+    "manufacturer": "Redragon",
+    "url": "https://www.redragonzone.com/products/redragonk556",
+    "maintainer": "Xinayder",
+    "bootloader": "sn32-dfu",
+    
+    "processor": "SN32F248BF",
+    "usb": {
+        "vid": "0x0C45",
+        "pid": "0x5004",
+        "device_version": "1.0.0",
+        "force_nkro": true
+    },
+
+    "features": {
+        "audio": false,
+        "backlight": false,
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": false,
+        "nkro": true,
+        "rgb_matrix": true
+    },
+
+    "debounce": 5,
+    "diode_direction": "ROW2COL",
+    "matrix_pins": {
+        "cols": [ "A0",  "A1",  "A2",  "A3", "A4", "A5",  "A6",  "A7",  "A8", "A9", "A10", "A11", "A12", "A13", "A14", "B0", "B1", "B2", "B3", "B4", "B5" ],
+        "rows": [ "C15", "D11", "D10", "D9",  "D8", "D7" ]
+    },
+    
+    "indicators": {
+        "num_lock": "B13",
+        "caps_lock": "B14",
+        "scroll_lock": "B15",
+        "on_state": 0
+    },
+    
+    "rgb_matrix": {
+        "animations": {
+            "band_val" : true,
+            "breathing" : true,
+            "cycle_left_right" : true,
+            "cycle_pinwheel" : true,
+            "cycle_spiral" : true,
+            "digital_rain" : true,
+            "gradient_left_right" : true,
+            "gradient_up_down" : true,
+            "raindrops" : true,
+            "solid_reactive_simple" : true,
+            "splash" : true,
+            "typing_heatmap" : true
+        },
+        "driver": "SN32F24xB",
```

```suggestion
        "driver": "sn32f24xb",
```
see https://github.com/SonixQMK/qmk_firmware/commit/d62cda8c08685b902d9e3b02276398a8bc61d56c